proto/prehog/v1alpha/teleport.proto

```diff
@@ -1452,6 +1483,12 @@ message SubmitEventRequest {
     UIDiscoverKubeEKSEnrollEvent ui_discover_kube_eks_enroll_event = 84;
 
     UIDiscoverCreateAppServerEvent ui_discover_create_app_server_event = 85;
+
+    AccessGraphGitlabScanEvent access_graph_gitlab_scan = 86;
```

Comment:
FYI, we should also add AWS integration here.

Reply:
I have a separate PR for that because events are emitted by the discovery service